chart/Makefile

```diff
@@ -0,0 +1,54 @@
+RELEASE_NAMESPACE=--namespace kube-system
+
+all: lint snapshots templates
+
+# ----- Snapshots -----
+
+default-snapshot: .snapshots/default.yaml
+	helm template hcloud-hccm . \
+		$(RELEASE_NAMESPACE) | \
+		grep -v helm.sh/chart \
+		> .snapshots/default.yaml
+
+full-snapshot: .snapshots/full.daemonset.yaml
+	helm template hcloud-hccm . \
+		$(RELEASE_NAMESPACE) \
+		-f .snapshots/full.values.daemonset.yaml \
```

Comment:
Weird, I think this values file should be moved under `chart/full-values.yaml`

Then you could run a snapshot for all the values files present in the `chart/` directory (using wildcard or similar)